shared/test/test_redis_table.rb

```diff
@@ -163,9 +167,11 @@ def test_delete_many
   end
 
   def test_expiration
+    stop_time
+
     # Test with a 1-second expire time
-    expire_time = 0.01
-    margin_time = 0.002
+    expire_time = 1
+    margin_time = 0.2
```

Comment:
[I tuned these values down](https://github.com/code-dot-org/code-dot-org/commit/8ff7b67c010dc2617fac1c3e506d9fcb4c97e96a) last August when we swapped in the FakeRedis gem.  Shame on me, this broke the test for real Redis because the EXPIRE command can only accept integer values.  Making the expire time an integer again fixes the test on real Redis, and now doesn't slow the test down on FakeRedis thanks to Timecop!